config-service-impl/src/test/java/org/hypertrace/config/service/ConfigServiceGrpcImplTest.java

```diff
@@ -232,14 +233,7 @@ void deleteConfigs() throws IOException {
     verify(configStore, times(1))
         .deleteConfigs(
             eq(Set.of(getConfigResourceContext(context1), getConfigResourceContext(context2))));
-    verify(responseObserver, times(1))
-        .onNext(
-            DeleteConfigsResponse.newBuilder()
-                .addAllDeletedConfigs(
-                    List.of(
-                        buildContextSpecificConfig(context1, config1, 10L, 20L),
-                        buildContextSpecificConfig(context2, config2, 10L, 20L)))
-                .build());
+    verify(responseObserver, times(1)).onNext(argThat(this::matchDeletedResponse));
```

Comment:
this test was flaky - was failing intermittently, so fixed it (though unrelated to this PR)